deployments/charts/service/values.yaml

```
@@ -1367,6 +1548,299 @@ services:
     # maxRequests: 100
     # routes: [...]
 
+  ## UI service configuration for the OSMO web frontend
+  ##
+  ui:
```

YAMLlint (1.38.0)

[error] duplication of key "ui" in mapping

(key-duplicates)